AnnoyancesFilter/Popups/sections/subscriptions_specific.txt

```diff
@@ -51,6 +51,7 @@ gaugau.futabanet.jp,newscientist.com,jiji.com,edp24.co.uk,technologyreview.com,n
 !
 ! SECTION: Subscriptions - Regular rules
 !
+imf.org##button[onclick="toggleSharePopup();"]
```

Comment:
@Alex-302 Shouldn't it be AdGuard Social Media filter?

Reply:
Yes, will move)